Bottom: 1c6f0df43a313d257b658f5aaac7cd035b123a55
Top:    45d8dec5b93942f532dc4bf19de1c84784d25831
Author: Parth Shah <parth@linux.ibm.com>
Date:   2020-05-07 19:00:34 +0530

Refresh of sched-idle-disable-idle-call

---

diff --git a/kernel/sched/idle.c b/kernel/sched/idle.c
index 9b16fa71f114..85d72a6e2521 100644
--- a/kernel/sched/idle.c
+++ b/kernel/sched/idle.c
@@ -231,7 +231,6 @@ static void cpuidle_idle_call(void)
 static void do_idle(void)
 {
 	int cpu = smp_processor_id();
-
 	/*
 	 * If the arch has a polling bit, we maintain an invariant:
 	 *
